src/utils.ts

```diff
@@ -315,8 +338,12 @@ export async function* syncToLatestBlock(): AsyncGenerator<StatusEvent> {
       for (const key of Object.keys(uniqueTxs)) {
       const preparedTx: PreparedTx = prepareTx(uniqueTxs[key]);
 
+
       try {
-        const sendTxResponse: ApiResponse = await sendTx(preparedTx);
+        const sendTxResponse: ApiResponse = await sendTx({
+          ...preparedTx,
+          height: preparedBlock.height,
```

Comment:
Why did you add this key?

Reply:
This was added by mistake on this branch, it is part of some changes I am doing to the sync daemon to handle reorgs

Removed it from last commit and force pushed it.